pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -121,6 +121,8 @@ func Pod(source *v1alpha1.CatalogSource, name string, image string, saName strin
 			Annotations:  annotations,
 		},
 		Spec: v1.PodSpec{
+			// TODO: Remove this before merging
```

Comment:
This is just so that I can easily run the catalog sources on kind and still talk to them via running/debugging the operator in GoLand. It will go away.